test/run-helm-tests.sh

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+# This runs helm tests for a release using `helm test` command
+K8S_VERSION=${KUBERNETES_VERSION:-v1.18.2}
```

Comment:
```suggestion
K8S_VERSION=${KUBERNETES_VERSION:-v1.20.2}
```

Reply:
Updated the K8s version with latest push.